relentless/simulate/hoomd.py

```diff
@@ -230,15 +230,15 @@ class InitializeRandomly(Initialize):
 
     Parameters
     ----------
+    seed : int
+        The seed to randomly initialize the particle locations.
     neighbor_buffer : float
         Buffer width.
-    seed : int
-        The seed to randomly initialize the particle locations (defaults to `None`).
     options : kwargs
         Options for random initialization.
 
     """
-    def __init__(self, neighbor_buffer, seed=None, **options):
+    def __init__(self, seed, neighbor_buffer, **options):
```

Comment:
We don't use `**options` anywhere here. Is there a reason we needed this, or can it be removed?